client/src/clusterStatus/NotConnected.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from "react";
+import { List, Modal } from "semantic-ui-react";
+
+const NotConnected: React.SFC = () => (
+        <Modal.Content>
+                <List>
+                    <List.Item>
+                        <List.Content>
+                            Not Connected
+                        </List.Content>
```

Comment:
should we add something more like a help/ link to docs?

Reply:
Not sure, when we are in the "not connected" state, the only thing shown is the "connect to cluster" / "create cluster" dialog anyways, right?